Models/c172p.xml

```diff
@@ -7478,10 +7521,18 @@
         <type>select</type>
         <object-name>TiedownHotSpotLeft</object-name>
         <condition>
-            <property>/sim/model/c172p/securing/tiedownL-addable</property>
-            <not>
-                <property>/sim/current-view/internal</property>
-            </not>
+            <or>
```

Comment:
if you have (A and B) or (A and C), then you can simplify this to this to A and (B or C). In this case A is `<property>/sim/model/c172p/securing/tiedownL-addable</property>`.